modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_files_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    module Veterans
+      class IntentToFilesController < ClaimsApi::V2::ApplicationController
```

Comment:
this is a nitpick out of curiosity, but any reason for opting for plural over singular on `File(s)`? I.e., `IntentToFilesController` over `IntentToFileController`

Reply:
Lee and I had talked about this at one point and we felt that pluralizing the route (and therefore the controller name) felt right.  Doesn't mean that it can't be changed of course.

Reply:
ahhh, ok. thinking about in the context of multiple ITFs for the same veteran makes more sense. I don't mind that, then